crates/cxx-qt-gen/test_inputs/properties.rs

```diff
@@ -20,4 +20,14 @@ mod ffi {
         #[qproperty(i32, final_prop, READ, WRITE, FINAL)]
         type MyObject = super::MyObjectRust;
     }
+
+    unsafe extern "RustQt" {
+        fn my_getter(self: &MyObject) -> i32;
+
+        fn my_setter(self: Pin<&mut MyObject>, value: i32);
+
+        fn myOnChanged(self: Pin<&mut MyObject>);
```

Comment:
This should have `#[qsignal]` on it.

Reply:
This I think is acc a bug I need to fix, as using qsignal means the code won't compile as the lookup searches for a method not a signal, and all the flags are using the same logic for custom functions

Reply:
Right a notify should be a signal, others like read, write, reset should be methods.

Reply:
Guess this should ideally be the snake case variant, which then means it'd check if it picks up the automatic camelCase conversion?

eg use

```
#[qproperty(i32, custom_on_changed_prop, READ, WRITE, NOTIFY = my_on_changed)]

...

       #[qsignal]
        fn my_on_changed(self: Pin<&mut MyObject>);
```

Then we should see `my_on_changed` on the Rust side and `myOnChanged` in the C++ generation.

Reply:
as then we have changes for everything

- rust_name (getter one)
- cxx_name (setter one)
- automatic camel conversion (notify one)
- already camel in Rust no conversion (reset one)

Reply:
Nice looks good :-)

Reply:
sorted